fractal/cmd/_task.py

```diff
@@ -143,12 +143,13 @@ async def patch_task(
     meta_file: Optional[str] = None,
 ) -> BaseInterface:
 
-    try:
-        task_id = int(task_id_or_name)
-        if version:
-            logging.warning("Task Version is ignored because Task ID provided")
-    except ValueError:
-        task_name = task_id_or_name
+    if task_id and version:
+        print(
```

Comment:
It's probably better to use `logging.error(..)`

Reply:
Actually: why did we move from a warning to a `sys.exit`? I think the warning was reasonable.